src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx

```diff
@@ -19,24 +19,34 @@ export const PriceDatabaseBenefits: React.FC = () => {
       <GridColumns mt={4} gridRowGap={[2, 0]}>
         <Column span={12}>
           <Text as="h1" variant={["xl", "xxl"]}>
-            Auction records from 340,000
+            Auction records from 300,000
             <br />
             artists—and counting
           </Text>
         </Column>
       </GridColumns>
 
       <Section
-        title="The largest publicly available art market database"
-        text="With millions of auction records, our growing database encompasses over 900 auction houses with pricing that dates back to 1986. Get real-time records from history-making auction houses like Christie's, Sotheby's, Phillips, Bonhams, and more."
+        title="Get in-depth art market data"
+        description={
+          <Text variant="sm">
+            Browse millions of current and historical results from leading
+            auction houses across the globe.
+          </Text>
```

Comment:
What is the reason for this change?

Reply:
@MounirDhahri 👀?

Reply:
we just spoke of this during the standup.

Reply:
@olerichter00 yes, we will add the value prop 4 once we get the image for value prop 4 from the Brand team

Reply:
But I agree, it makes sense to revert this until we need it